components/support/test/src/main/java/mozilla/components/support/test/robolectric/Fragments.kt

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.support.test.robolectric
+
+import androidx.fragment.app.Fragment
+import androidx.fragment.app.FragmentActivity
+import org.robolectric.Robolectric
+
+/**
+ * Set up an added [Fragment] to a [FragmentActivity] that has been initialized to a resumed state.
+ *
+ * @param fragmentTag the name that will be used to tag the fragment inside the [FragmentManager].
+ * @param fragmentFactory a lambda function that returns a Fragment that will be added to the Activity.
+ *
+ * @return The same [Fragment] that was returned from [fragmentFactory] after it got added to the
+ * Activity.
+ */
+inline fun <T : Fragment> createAddedFragment(fragmentTag: String = "test", fragmentFactory: () -> T): T {
```

Comment:
I think a name with "test" in it would make it clear that this is a helper for adding fragments in tests.

Reply:
Thanks for your review, but it is not clear to me which part I should change, the `createAddedFragment` to `createAddedTestFragment` or change the `fragmentTag` to something else?

Reply:
Sorry that I wasn't clear about my comment.  Yes, I meant to change the function name to "createAddedTestFragment".  Thanks,